examples/fetch/src/post.rs

```diff
@@ -70,21 +70,18 @@ pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
 
 pub fn view(model: &Model) -> Node<Msg> {
     form![
-        ev(Ev::Submit, |_| Msg::Submit),
+        ev(Ev::Submit, |event| {
+            event.prevent_default();
+            Msg::Submit
+        }),
         label![
             "Name",
             input![
                 attrs! {At::Value => model.form.name},
                 input_ev(Ev::Input, Msg::NameChanged),
             ]
         ],
-        button![
-            "Submit",
-            ev(Ev::Click, |event| {
-                event.prevent_default();
-                Msg::Submit
-            })
-        ],
+        button!["Submit",],
```

Comment:
Thank you mate ! 

I have ran `cargo fmt` but it did not format. I guess I ll do myself :)

Reply:
So now we have ` button!["Submit"],`

That is good , right ?

Reply:
Yes, that is correct. rustfmt knows about `vec![]` because it is standard so it is able to special case it. rustfmt knows nothing about `button!` so the formatting has to be done a little more manually.

Reply:
Thank you for your explanation :) This does make sens ! I close the discussion then !